File: Coursera Courses/Specializations/Programming in Java: A Hands-on Introduction/Object-Oriented Java: Inheritance and Encapsulation/Week 1/LabChallenge.java
//add class definitions below this line
    
class Person {
  private String name;
  private int age;
  private String occupation;

  public Person(String n, int a, String o) {
    name = n;
    age = a;
    occupation = o;
  }
  
  public String getName() {
    return name;
  }

  public void setName(String newName) {
    name = newName;
  }

  public int getAge() {
    return age;
  }

  public void setAge(int newAge) {
    age = newAge;
  }
  
  public String getOccupation() {
    return occupation;
  }

  public void setOccupation(String newOccupation) {
    occupation = newOccupation;
  }
}
  
//add class definitions above this line

public class LabChallenge {
  public static void main(String[] args) {
    
    //add code below this line

    Person test = new Person("Citra Curie", 16, "student");
    
    System.out.println(test.getName());
    test.setName("Rowan Faraday");
    System.out.println(test.getName());
    
    System.out.println(test.getAge());
    test.setAge(18);
    System.out.println(test.getAge());
    
    System.out.println(test.getOccupation());
    test.setOccupation("plumber");
    System.out.println(test.getOccupation());

    //add code above this line
  }
}

File: Coursera Courses/Specializations/Programming in Python: A Hands-on Introduction/Object-Oriented Python: Inheritance and Encapsulation/Week 2/lab_challenge.py
class Person:
  def __init__(self, name, age, occupation):
    self._name = name
    self._age = age
    self._occupation = occupation
    
  def get_name(self):
    return self._name
  
  def set_name(self, new_name):
    self._name = new_name
    
  def get_age(self):
    return self._age
  
  def set_age(self, new_age):
    self._age = new_age
    
  def get_occupation(self):
    return self._occupation
  
  def set_occupation(self, new_occupation):
    self._occupation = new_occupation
